docs/en/ingest-management/release-notes/release-notes-8.12.asciidoc

```diff
@@ -0,0 +1,244 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.12.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.12.0 relnotes
+
+[[release-notes-8.12.0]]
+== {fleet} and {agent} 8.12.0
+
+Review important information about {fleet-server} and {agent} for the 8.12.0 release.
+
+[discrete]
+[[security-updates-8.12.0]]
+=== Security updates
+
+{agent}::
+* Update Go version to 1.20.12. {agent-pull}3885[#3885]
+
+[discrete]
+[[known-issues-8.12.0]]
+=== Known issues
+
+[[known-issue-4084]]
+.`elastic-agent enroll` command incorrectly reports failure
+[%collapsible]
+====
+
+*Details*
+
+When you run the <<elastic-agent-enroll-command,`elastic-agent enroll`>> command for an RPM or DEB {agent} package, a `Retarting agent daemon` message appears in the command output, followed by a `Restart attempt failed` error.
+
+*Impact* +
+
+The error does not mean that the enrollment failed. The enrollment actually succeeded. You can ignore the `Restart attempt failed` error and continue by running the following commands, after which {agent} should successfully connect to {fleet}:
+
+[source,console]
+----
+sudo systemctl enable elastic-agent 
+sudo systemctl start elastic-agent
+----
+
+====
+
+[discrete]
+[[new-features-8.12.0]]
+=== New features
+
+The 8.12.0 release Added the following new and notable features.
+
+{fleet}::
+* Add support for preconfigured output secrets (Scrypt edition). ({kibana-pull}172041[#172041])
+* Add UI components to create and edit output secrets. ({kibana-pull}169429[#169429])
+* Add support for remote ES output. ({kibana-pull}169252[#169252])
+* Add the ability to specify secrets in outputs. ({kibana-pull}169221[#169221])
+* Add an integrations configs tab to display input templates. ({kibana-pull}168827[#168827])
+* Add a {kib} task to publish Agent metrics. ({kibana-pull}168435[#168435])
+
+{agent}::
+//* Add support for processors in hints-based Kubernetes autodiscover. {agent-pull}3107[#3107] {agent-issue}2959[#2959]
```

Comment:
```suggestion
* Add support for processors in hints-based Kubernetes autodiscover. {agent-pull}3107[#3107] {agent-issue}2959[#2959]
```

Reply:
Actually I just removed this one (I copied it in from a previous release just as a model for the PR/issue link format).